_data/apps.yml

```diff
@@ -1101,3 +1101,13 @@
     - "database"
     - "design"
   icon: "collectie.png"
+-
+  name: "Calculator"
+  description: "A Basic Calculator"
+  website: "https://drive.google.com/open?id=0B4vBBfXB3KipdHBTU0IwdDllaGc"
```

Comment:
My application doesn't have any website and i can't upload the application to repository because it only allows me to upload 25mb and not more